plugins/sracq.py

```diff
@@ -113,12 +113,8 @@ def __init__(self, microscope, main_app):
 
         self.addMenu("Acquisition/Super-resolution...", self.start)
 
-        # light.power has a maximum range corresponding to the maximum source,
-        # and other sources are actually proportionally scaled, which is very
-        # confusing. So make it a bit more obvious that it's just a ratio.
-        pwr = 100 * self.light.power.value / self.light.power.range[1]
-        self.power = model.FloatContinuous(pwr, range=(0, 100), unit="%",
-                                           setter=self._set_power)
+        # light.power has a maximum range corresponding to each source
+        self.power = self.light.power
```

Comment:
I removed this power conversion part but I'm not sure what is the use of self.power in this plugin now. Should it be removed or the ratio is used somewhere?
@pieleric

Reply:
Yeah I noticed the plugin doesn't use the stream directly, so should I remove `power` attribute from the plugin entirely @pieleric ? 

And also switching off the light power?
```
# Switch off laser
self.light.power.value = self.light.power.range[0]
```

Reply:
Yes, you should remove `.power` from the plugin class.
To switch off the power (l 322), your proposal seems fine.